spiceaidocs/docs/machine-learning/index.md

```diff
@@ -8,11 +8,14 @@ pagination_prev: null
 
 :::warning[Early Preview]
 
-The Spice ML runtime is in its early preview phase and is subject to modifications.
+Machine Learning (ML) is in preview and is subject to modifications.
 
 :::
 
-Machine learning models can be added to the Spice runtime similarly to datasets. The Spice runtime will load it, just like a dataset. 
+ML models can be defined similarly to datasets. The runtime will load the model for inference.
```

Comment:
Put a link to datasets.  Add a note that understanding datasets is a prequisite for this portion of the documents.